src/Logic.hs

```diff
@@ -829,3 +916,34 @@ pullRequestIdToText (PullRequestId prid) = Text.pack $ show prid
 
 testBranch :: ProjectConfiguration -> PullRequestId -> Git.Branch
 testBranch config pullRequestId = Git.Branch $ Config.testBranch config <> "/" <> pullRequestIdToText pullRequestId
+
+-- | Textual rendering of a list of 'PullRequestId's
+--
+-- >>> prettyPullRequestIds [PullRequestId 12, PullRequestId 60, PullRequestId 1337]
+-- "#12, #60 and #1337"
```

Comment:
Unfortunately not, we would have to set up and use [the DocTest package](https://hackage.haskell.org/package/DocTest).  This could be a nice addition at some point, I'll try to create an issue.